### PR TITLE
Fallen / Banished / Rare KillTasks added

### PR DESCRIPTION
**Pull request by sLackeR-hub**
Updated Belinda per [http://acpedia.org/wiki/Belinda_du_Loc](url) per retail with KillTasks and Turn-Ins.

**Added**
**Items-**
Enhanced Basalt Blade
Enhanced Crystal Sword
Enhanced Dark Sorcerer's Phylactery
Enhanced Energy Crown
Enhanced Fetid Dirk
Enhanced Gauntlets of Marksmanship
Enhanced Obsidian Axe
Enhanced Osseous Mace
Enhanced Robe of the Tundra
Enhanced Serpent's Fang
Enhanced Skeletal Atlatl
Enhanced Soul Staff

**Updated**
23535 Obsidian Axe (Icon)

### CODE REVIEW

**Review by DridsGhost — APPROVED** [review]
Approved.

**Review by DridsGhost — COMMENTED** [review]
Found some discrepancies with weapons. More testing required.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Creature/Human/33966_Belinda_du_Loc.es [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+							- AwardNoShareXP: 35,000,000
+							- EraseQuest: KilltaskFallenCreature_0609
+							- StampQuest: KilltaskFallenCreatureWait_0609
+							- Goto: Rare
+						QuestFailure:
+							- Tell: I would like you to experience the hunting of the elusive fallen creatures. Return to me when you have located and killed 10 of them.
+							- SetQuestCompletions: KilltaskFallenCreature_0609, 0
+							- Goto: Rare
+
+GotoSet: Rare
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Creature/Human/33966_Belinda_du_Loc.es:72]: This needs to be updated to reflect 50 kills required.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Creature/Human/33966_Belinda_du_Loc.es [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+    - TurnToTarget
+    - Tell: Belinda du Loc tells you, "I have enhanced this item using the essence of the creature that once wielded it, put it to good use. If you do not want it give it back and I will reward you alternatively.
+    - Give: 33940
+
+Give: 33940
+    - TurnToTarget
+    - Tell: I see you didn't want to keep this, that's a shame.
+    - AwardLevelProportionalXP: 1%, 500,000
+
+Give: Dark Sorcerer's Phylactery (12016)
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Creature/Human/33966_Belinda_du_Loc.es:245]: There appears to be more than one of these weapons in the 16PY. Please review these additional weapons.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (33950,   1,          4) /* ItemType - Clothing */
+     , (33950,   3,          9) /* PaletteTemplate - Grey */
+     , (33950,   4,      98048) /* ClothingPriority - OuterwearUpperLegs, OuterwearLowerLegs, OuterwearChest, OuterwearAbdomen, OuterwearUpperArms, OuterwearLowerArms, Head, Feet */
+     , (33950,   5,        500) /* EncumbranceVal */
+     , (33950,   9,      32513) /* ValidLocations - HeadWear, Armor */
+     , (33950,  16,          1) /* ItemUseable - No */
+     , (33950,  19,       6000) /* Value */
+     , (33950,  28,        500) /* ArmorLevel */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:14]: Armor level is incorrect. Wiki lists 150

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+     , (33950,  93,       1044) /* PhysicsState - Ethereal, IgnoreCollisions, Gravity */
+     , (33950, 106,        200) /* ItemSpellcraft */
+     , (33950, 107,       1246) /* ItemCurMana */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:17]: Update ItemCurMana

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (33950,   1, 'Enhanced Robe of the Tundra') /* Name */
+     , (33950,  16, 'A robe crafted from a mattekar hide.  It has some natural padding in it that makes it stronger and more resistant to damage. This robe has been enhanced by Belinda du Loc.') /* LongDesc */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (33950,   1,   33554854) /* Setup */
+     , (33950,   3,  536870932) /* SoundTable */
+     , (33950,   6,   67108990) /* PaletteBase */
+     , (33950,   7,  268436263) /* ClothingBase */
+     , (33950,   8,  100674100) /* Icon */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:49]: Icons probably wrong?

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/Armor/33949 Enhanced Gauntlets of Marksmanship.sql [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (33949,   1,          2) /* ItemType - Armor */
+     , (33949,   4,      32768) /* ClothingPriority - Hands */
+     , (33949,   5,        450) /* EncumbranceVal */
+     , (33949,   9,         32) /* ValidLocations - HandWear */
+     , (33949,  16,          1) /* ItemUseable - No */
+     , (33949,  18,          1) /* UiEffects - Magical */
+     , (33949,  19,       6000) /* Value */
+     , (33949,  28,        325) /* ArmorLevel */
+     , (33949,  93,       1044) /* PhysicsState - Ethereal, IgnoreCollisions, Gravity */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/Armor/33949 Enhanced Gauntlets of Marksmanship.sql:15]: This item has no mana limit/tank etc...

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33942 Enhanced Serpent's Fang.sql [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+     , (33942,  44,         40) /* Damage */
+     , (33942,  45,         32) /* DamageType - Acid */
+     , (33942,  46,          2) /* DefaultCombatStyle - OneHanded */
+     , (33942,  47,          2) /* AttackType - Thrust */
+     , (33942,  48,         45) /* WeaponSkill - LightWeapons */
+     , (33942,  49,         30) /* WeaponTime */
+     , (33942,  51,          1) /* CombatUse - Melee */
+     , (33942,  93,       1044) /* PhysicsState - Ethereal, IgnoreCollisions, Gravity */
+     , (33942, 106,        275) /* ItemSpellcraft */
+     , (33942, 107,       1511) /* ItemCurMana */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33942 Enhanced Serpent's Fang.sql:22]: fix ItemCurMana

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33944 Enhanced Soul Staff.sql [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+     , (33944,  44,         55) /* Damage */
+     , (33944,  45,         64) /* DamageType - Electric */
+     , (33944,  46,          2) /* DefaultCombatStyle - OneHanded */
+     , (33944,  47,          6) /* AttackType - Thrust, Slash */
+     , (33944,  48,         46) /* WeaponSkill - FinesseWeapons */
+     , (33944,  49,         20) /* WeaponTime */
+     , (33944,  51,          1) /* CombatUse - Melee */
+     , (33944,  93,       1044) /* PhysicsState - Ethereal, IgnoreCollisions, Gravity */
+     , (33944, 106,        250) /* ItemSpellcraft */
+     , (33944, 107,       1377) /* ItemCurMana */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33944 Enhanced Soul Staff.sql:22]: Update ItemCurMana

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+     , (33946,  18,         32) /* UiEffects - Fire */
+     , (33946,  19,       2700) /* Value */
+     , (33946,  44,         31) /* Damage */
+     , (33946,  45,         16) /* DamageType - Fire */
+     , (33946,  46,          2) /* DefaultCombatStyle - OneHanded */
+     , (33946,  47,          1) /* AttackType - Punch */
+     , (33946,  49,         20) /* WeaponTime */
+     , (33946,  51,          1) /* CombatUse - Melee */
+     , (33946, 106,        250) /* ItemSpellcraft */
+     , (33946, 107,       1152) /* ItemCurMana */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql:19]: Update ItemCurMana

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql [hunk 10]
```diff
@@ -0,0 +1,59 @@
+DELETE FROM `weenie` WHERE `class_Id` = 33946;
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql:1]: This weapon is missing properties and cannot be used.

**Review by DridsGhost — COMMENTED** [review]
Suggest reviewing the weapon exchanges, appeared to be missing the dark sorcerers phylactery as found in 16PY.

Update Belinda's Rare killtask to reflect the required 50 and revalidate.

Minor updates on a few of the enhanced weapons.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql (hunk 3 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:14]: PCAP had it 500

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql (hunk 5 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:49]: PCAPd Icon

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql (hunk 3 above)
Comment (DridsGhost) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:14]: it was probably buffed

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql (hunk 5 above)
Comment (DridsGhost) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:49]: It was probably dyed black.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql (hunk 3 above)
Comment (DridsGhost) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:14]: Check out the shard data

https://github.com/Mag-nus/ACE-Shard-Retail/blob/master/Morningthaw/Item%20Mule%20III/0/872556D6%20Enhanced%20Robe%20of%20the%20Tundra.sql

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql (hunk 5 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:49]: Changed icon to original Robes icon.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql (hunk 4 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:17]: Updated.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql (hunk 3 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/33950 Enhanced Robe of the Tundra.sql:14]: Updated.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Creature/Human/33966_Belinda_du_Loc.es (hunk 2 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Creature/Human/33966_Belinda_du_Loc.es:245]: Should have all variations for turn-in.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/Armor/33949 Enhanced Gauntlets of Marksmanship.sql (hunk 6 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Clothing/Armor/33949 Enhanced Gauntlets of Marksmanship.sql:15]: Fixed.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33942 Enhanced Serpent's Fang.sql (hunk 7 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33942 Enhanced Serpent's Fang.sql:22]: Fixed.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33944 Enhanced Soul Staff.sql (hunk 8 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33944 Enhanced Soul Staff.sql:22]: Fixed.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql (hunk 9 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql:19]: Fixed.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql (hunk 10 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql:1]: Still seeing issues with this weapon, not sure what is going on with it.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33940 Enhanced Crystal Sword.sql [hunk 11]
```diff
@@ -27,19 +28,17 @@ VALUES (33940,   1,          1) /* ItemType - MeleeWeapon */
...
-VALUES (33940,   5, -0.032999999821186066) /* ManaRate */
+VALUES (33940,   5,  -0.033) /* ManaRate */
      , (33940,  21,       0) /* WeaponLength */
      , (33940,  22,     0.5) /* DamageVariance */
      , (33940,  26,       0) /* MaximumVelocity */
-     , (33940,  29, 1.1200000047683716) /* WeaponDefense */
-     , (33940,  39, 1.100000023841858) /* DefaultScale */
-     , (33940,  62, 1.1200000047683716) /* WeaponOffense */
-     , (33940,  63,       1) /* DamageMod */
-     , (33940, 156, 0.05000000074505806) /* ProcSpellRate */;
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33940 Enhanced Crystal Sword.sql:42]: Shouldnd this have a chance to cast on strike according to the Wiki?

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33942 Enhanced Serpent's Fang.sql [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+INSERT INTO `weenie_properties_float` (`object_Id`, `type`, `value`)
+VALUES (33942,   5,  -0.033) /* ManaRate */
+     , (33942,  21,       0) /* WeaponLength */
+     , (33942,  22,    0.66) /* DamageVariance */
+     , (33942,  26,       0) /* MaximumVelocity */
+     , (33942,  29,     1.1) /* WeaponDefense */
+     , (33942,  62,    1.15) /* WeaponOffense */
+     , (33942,  63,       1) /* DamageMod */
+     , (33942, 136,       1) /* CriticalMultiplier */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33942 Enhanced Serpent's Fang.sql:42]: May want to bump this up to 2 on the cirticalMultiplier, most things in 16PY are 2-4.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33945 Enhanced Fetid Dirk.sql [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+     , (33945,  22, True ) /* Inscribable */;
+
+INSERT INTO `weenie_properties_float` (`object_Id`, `type`, `value`)
+VALUES (33945,   5,  -0.025) /* ManaRate */
+     , (33945,  22,     0.3) /* DamageVariance */
+     , (33945,  26,       0) /* MaximumVelocity */
+     , (33945,  29,    1.08) /* WeaponDefense */
+     , (33945,  62,    1.08) /* WeaponOffense */
+     , (33945,  63,       1) /* DamageMod */
+     , (33945, 136,       1) /* CriticalMultiplier */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33945 Enhanced Fetid Dirk.sql:41]: CriticalMultiplier should be between 2-4

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33945 Enhanced Fetid Dirk.sql [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (33945,   1,   33558184) /* Setup */
+     , (33945,   3,  536870932) /* SoundTable */
+     , (33945,   6,   67114156) /* PaletteBase */
+     , (33945,   8,  100674031) /* Icon */
+     , (33945,  22,  872415275) /* PhysicsEffectTable */
+     , (33945,  37,         44) /* ItemSkillLimit - HeavyWeapons */;
+
+INSERT INTO `weenie_properties_spell_book` (`object_Id`, `spell`, `probability`)
+VALUES (33945,    61,      2)  /* Acid Stream IV */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33945 Enhanced Fetid Dirk.sql:57]: This weapon should have a cast on strike Proc and the spell should be Acid Stream

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql (hunk 10 above)
Comment (DridsGhost) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql:1]: Use the updated weenie I sent you for this weapon to resolve the issue.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+     , (33946,  22, True ) /* Inscribable */;
+
+INSERT INTO `weenie_properties_float` (`object_Id`, `type`, `value`)
+VALUES (33946,   5,  -0.025) /* ManaRate */
+     , (33946,  22,     0.3) /* DamageVariance */
+     , (33946,  26,       0) /* MaximumVelocity */
+     , (33946,  29,    1.15) /* WeaponDefense */
+     , (33946,  62,     1.1) /* WeaponOffense */
+     , (33946,  63,       1) /* DamageMod */
+     , (33946, 147,       1) /* CriticalFrequency */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql:42]: Make sure to update the CriticalFrequency to below 1. Most things appear to have between 0.15 - 0.3

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/87143 Enhanced Obsidian Axe.sql [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+
+INSERT INTO `weenie_properties_float` (`object_Id`, `type`, `value`)
+VALUES (87143,   5,  -0.025) /* ManaRate */
+     , (87143,  21,    0.95) /* WeaponLength */
+     , (87143,  22,     0.5) /* DamageVariance */
+     , (87143,  26,       0) /* MaximumVelocity */
+     , (87143,  29,     1.1) /* WeaponDefense */
+     , (87143,  39,     1.1) /* DefaultScale */
+     , (87143,  62,     1.1) /* WeaponOffense */
+     , (87143,  63,       1) /* DamageMod */;
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/87143 Enhanced Obsidian Axe.sql:47]: DamageMod only appears on ranged weapons, please review the weapons in the PR and remove.

Comment (DridsGhost) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/87143 Enhanced Obsidian Axe.sql:47]: According to the wiki this weapon should have  Biting Strike, Resistance Cleaving: Fire

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MissileLauncher/87141 Enhanced Skeletal Atlatl.sql [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+VALUES (87141,   1, 'Enhanced Skeletal Atlatl') /* Name */
+     , (87141,  16, 'The radius and ulna of a Lord of Decay, twisted together into an atlatl and enhanced by Belinda du Loc.') /* LongDesc */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (87141,   1,   33558186) /* Setup */
+     , (87141,   3,  536870932) /* SoundTable */
+     , (87141,   6,   67111919) /* PaletteBase */
+     , (87141,   8,  100674030) /* Icon */
+     , (87141,  22,  872415275) /* PhysicsEffectTable */
+     , (87141,  37,         12) /* ItemSkillLimit - ThrownWeapon */;
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MissileLauncher/87141 Enhanced Skeletal Atlatl.sql:59]: ItemSkillLimit  not needed, only used in summoning as far as I know.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql (hunk 10 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql:1]: Fixed with yours.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33940 Enhanced Crystal Sword.sql (hunk 11 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33940 Enhanced Crystal Sword.sql:42]: Fixed, not sure how it was removed.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33942 Enhanced Serpent's Fang.sql (hunk 12 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33942 Enhanced Serpent's Fang.sql:42]: Fixed.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33945 Enhanced Fetid Dirk.sql (hunk 13 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33945 Enhanced Fetid Dirk.sql:41]: Fixed.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql (hunk 15 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33946 Enhanced Basalt Blade.sql:42]: Fixed with your copy.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MissileLauncher/87141 Enhanced Skeletal Atlatl.sql (hunk 17 above)
Comment (sLackeR-hub) [reply Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MissileLauncher/87141 Enhanced Skeletal Atlatl.sql:59]: Skill limit is for activation I believe, I could be wrong.

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33944 Enhanced Soul Staff.sql [hunk 18]
```diff
@@ -0,0 +1,60 @@
...
+
+INSERT INTO `weenie_properties_float` (`object_Id`, `type`, `value`)
+VALUES (33944,   5,  -0.025) /* ManaRate */
+     , (33944,  21,       0) /* WeaponLength */
+     , (33944,  22,    0.25) /* DamageVariance */
+     , (33944,  26,       0) /* MaximumVelocity */
+     , (33944,  29,     1.1) /* WeaponDefense */
+     , (33944,  62,     1.1) /* WeaponOffense */
+     , (33944,  63,       1) /* DamageMod */
+     , (33944, 157,       1) /* ResistanceModifier */;
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/MeleeWeapon/33944 Enhanced Soul Staff.sql:42]: Missing
156,    0.05) /* ProcSpellRate */

File: Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Caster/33953 Enhanced Dark Sorcerer's Phylactery.sql [hunk 19]
```diff
@@ -0,0 +1,47 @@
...
+VALUES (33953,   1, 'Enhanced Dark Sorcerer''s Phylactery') /* Name */
+     , (33953,  16, 'An orb with some sort of dark figure within. Gazing into its depths, you see the wretched face of a Dark Sorcerer, an undead from the Vesayen Islands. This orb has been enhanced by Belinda du Loc to unlock a greater magnitude of its true nature.') /* LongDesc */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (33953,   1,   33557338) /* Setup */
+     , (33953,   3,  536870932) /* SoundTable */
+     , (33953,   6,   67111919) /* PaletteBase */
+     , (33953,   8,  100674094) /* Icon */
+     , (33953,  22,  872415275) /* PhysicsEffectTable */
+     , (33953,  28,       2178) /* Spell - Decrepitude's Grasp */
```
Comment (DridsGhost) [inline Database/Patches/2006-09-ComeWhatFollows/9 WeenieDefaults/Caster/33953 Enhanced Dark Sorcerer's Phylactery.sql:39]: Probably should add 
 27, 1073742049) /* UseUserAnimation - UseMagicWand */

**Pull request merged**